docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -2062,6 +2093,103 @@ Override the default architecture variant of the container image.
 
 This is equivalent to the Podman `--variant` option.
 
+## Login units [Login]
+
+Login files are named with a `.login` extension and contain a section `[Login]` describing the
+login command. The generated service is a one-time command that logs into a registry.
+
+Using login units allows pulling images from private registries without having to login manually.
+
+There is only one required key, `Registry`, which defines the URL of the registry to log into.
+
+To avoid password leaking, Quadlet will set the permissions of the generated service file as 0600.
+It is recommended to do the same for `.login` unit files.
+
+Valid options for `[Login]` are listed below:
+
+| **[Login] options**                             | **podman login**                                                     |
+|-------------------------------------------------|----------------------------------------------------------------------|
+| AuthFile=/etc/registry/auth\.json               | --authfile=/etc/registry/auth\.json                                  |
+| CertDir=/etc/certs                              | --cert-dir=/etc/certs                                                |
+| LogoutOnStop=true                               | Add ExecStopPost to log out of the registry when the unit is stopped |
+| Password=mypassword                             | --password-stdin and set StandardInputText                           |
```

Comment:
Can we just… not do that?

Reply:
I don't think Quadlet should block users from doing what podman allows them to